api/envoy/service/status/v3/csds.proto

```diff
@@ -99,8 +99,11 @@ message PerXdsConfig {
   // Client config status is populated by xDS clients. Will not be present if
   // the CSDS server is an xDS server. No matter what the client config status
   // is, xDS clients should always dump the most recent accepted xDS config.
+  //
+  //   This field is deprecated. Use :ref:`ClientResourceStatus
+  // <envoy_api_msg_admin.v3.ClientResourceStatus>` instead.
   ClientConfigStatus client_status = 7
-      [(udpa.annotations.field_migrate).oneof_promotion = "status_config"];
+      [deprecated = true, (udpa.annotations.field_migrate).oneof_promotion = "status_config"];
```

Comment:
We can probably also get rid of the "oneof_promition" attribute, both here and on the `status` field above.  If this field is going away, there's no need for the oneof in the next version anymore.

Reply:
Annotation removed.